renovate.json

```diff
@@ -78,7 +78,7 @@
       }
     },
     {
-      "packageNames": ["@unlock-protocol/unlock-js"],
+      "packageNames": ["@unlock-protocol/unlock-js", "@unlock-protocol/unlock-assets", "unlock-abi-0-1", "unlock-abi-0-2", "unlock-abi-0", "unlock-abi-1-0", "unlock-abi-1-1", "unlock-abi-1-2", "unlock-abi-1-3", "unlock-abi-7"],
```

Comment:
this is fine, but they do support wildcards as well via packagePatterns instead of packageNames: https://docs.renovatebot.com/configuration-options/#packagepatterns